src/database/model/rideSummary.ts

```diff
@@ -0,0 +1,24 @@
+import {Model, Relation} from '@nozbe/watermelondb';
+import {field} from '@nozbe/watermelondb/decorators';
+import {immutableRelation} from '@nozbe/watermelondb/decorators';
+import Ride from './ride';
+
+export default class RideSummary extends Model {
+  static table = 'RideSummary';
```

Comment:
I think the table name here isn't consistent with the others, it should be `ride_summary` no?